frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -338,15 +334,66 @@ def cond_fn():
         (len(control) if isinstance(control, Sized) else 1)
         != (len(control_values) if isinstance(control_values, Sized) else 1)
     ):
-        raise ValueError(
-            f"Length of the control_values ({len(control_values)}) must be None or equal "
-            f"to the lenght of control ({len(control)})"
-        )
+        raise ValueError("control_values should be the same length")
```

Comment:
I liked the previous error message more. It had more information. Can it be changed back?

Reply:
yeah, I agree.. I'll change it back!

Reply:
By the way I had to change the error message here to make it consistent with PL.